mobile/src/main/java/com/alexstyl/specialdates/search/ContactWithEventsSearch.java

```diff
@@ -28,13 +28,13 @@ public static ContactWithEventsSearch newInstance(Context context) {
         return new ContactWithEventsSearch(context, contactProvider, nameMatcher);
     }
 
-    ContactWithEventsSearch(Context context, ContactProvider contactProvider, NameMatcher nameMatcher) {
+    private ContactWithEventsSearch(Context context, ContactProvider contactProvider, NameMatcher nameMatcher) {
```

Comment:
How are you planning to test this (or any of the other classes) if it has a private constructor? :trollface: